app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
```

Comment:
Input validation is missing. Ensure that the input string is validated to contain exactly three elements, with 'cp' as the first element. This will help prevent errors and improve the robustness of your code.